code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -398,19 +398,18 @@ datum/gas_reaction/freonfire/react(datum/gas_mixture/air, datum/holder)
 
 //fusion: a terrible idea that was fun but broken. Now reworked to be less broken and more interesting. Again (and again, and again). Again!
```

Comment:
It's highly important to add another again here

Reply:
issue resolved, the PR is in a mergeable state now